vendor/github.com/openshift/api/route/v1/generated.proto

```
@@ -0,0 +1,468 @@
+
+// This file was autogenerated by go-to-protobuf. Do not edit it manually!
+
+syntax = "proto2";
+
+package github.com.openshift.api.route.v1;
+
+import "k8s.io/api/core/v1/generated.proto";
```

Buf (1.47.2)

import "k8s.io/api/core/v1/generated.proto": file does not exist

(COMPILE)